workflow/r/slim/v1/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM r-base:4.2.0
+
+RUN apt-get update 
+RUN apt-get install -y git unixodbc unixodbc-dev
+
+# Install aws-cli
+RUN wget https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.2.5.zip -O awscliv2.zip \
+  && unzip -q awscliv2.zip \
+  && ./aws/install \
+  && rm -rf aws awscliv2.zip
+
+# Install Redshift ODBC Driver
+RUN wget https://s3.amazonaws.com/redshift-downloads/drivers/odbc/1.4.27.1000/AmazonRedshiftODBC-64-bit-1.4.27.1000-1.x86_64.deb -O AmazonRedshiftODBC.deb \
+  && apt install ./AmazonRedshiftODBC.deb \
+  && rm ./AmazonRedshiftODBC.deb
+
+COPY odbcinst.ini /etc/odbcinst.ini
+COPY amazon.redshiftodbc.ini /opt/amazon/redshiftodbc/lib/64/amazon.redshiftodbc.ini
+
+ENV ODBCSYSINI=/etc
+ENV AMAZONREDSHIFTODBCINI=/opt/amazon/redshiftodbc/lib/64/amazon.redshiftodbc.ini
```

Comment:
Hey @saurabh0402 - Can we also set snowflake ini path to ensure consistency ?

Reply:
These are based on the `r-studio` workspace image - we only have the redshift ones there. I was wondering whether we need these files or if we can omit these? 🤔

Reply:
We won't be able to fill in all the tenant-specific details in the obdc.ini files anyway, so maybe we can omit these and let the user create one as needed or directly use the env variables to connect.

Reply:
Updated.